ui/src/message_popup/scaffolding_card.jsx

```diff
@@ -119,7 +119,7 @@ export default React.createClass({
         {showSlider &&
           <div>
             <div style={styles.optionTitle}>Session Length: {sessionLength} {sessionLength===1 ? "question" : "questions"}</div>
-            <Slider key={selectedIndicatorId} value={sessionLength} min={1} max={questions.length} step={1} onChange={this.onSliderChange}/>
```

Comment:
ah good, thanks for explaining!  That's a good way to use `key` to workaround, but also clearly a bug in the library with not updating based on the prop change.

I went and did the good open source citizen thing and submitted the fix upstream in https://github.com/callemall/material-ui/pull/4895, so hopefully they'll pull it in and in the meantime I'll add back your great solution.  Thanks!